GitOut/Features/Git/Log/GitLogPage.xaml

```diff
@@ -455,8 +455,17 @@
                         Style="{StaticResource MaterialButtonStyle}"
                         Command="{Binding FetchRemotesCommand}"
                       >
-                                                FETCH
-                                            </Button
+                        FETCH
+                      </Button
+                      >
+                      <Button
```

Comment:
Can we place it in a horizontal StackPanel instead?
<img width="237" height="229" alt="image" src="https://github.com/user-attachments/assets/252b8418-1880-4682-b6bc-cac30e2d0d96" />